docs/release_notes/0.75.0.md

```diff
@@ -0,0 +1,18 @@
+# Release 0.75.0
```

Comment:
shouldn't this be Release 0.74.0 ?

Reply:
my bad read the releases page wrong https://github.com/weaveworks/eksctl/releases 🙈  
ignore me :D